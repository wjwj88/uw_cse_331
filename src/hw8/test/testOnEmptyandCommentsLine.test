# this is to test echoing empty lines and 
# lines starting with #.

# works well

# terminate program
q