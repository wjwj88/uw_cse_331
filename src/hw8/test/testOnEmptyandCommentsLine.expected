Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # this is to test echoing empty lines and 
# lines starting with #.

# works well

# terminate program